snippets/usbip-native-sim/usbip-native-sim.conf

```diff
@@ -16,6 +16,6 @@ CONFIG_USBH_LOG_LEVEL_WRN=y
 CONFIG_UHC_DRIVER_LOG_LEVEL_WRN=y
 CONFIG_USBIP=y
 CONFIG_USBIP_LOG_LEVEL_WRN=y
-CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000000
+# CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000000
```

Comment:
@jfischer-no as you added this, please elaborate on why this is necessary. Manual testing shows native sim works fine without this config, however it breaks nRF clock drivers in bsim

Reply:
> @jfischer-no as you added this, could you please elaborate on why this is necessary. 

It changes minimum available timing granularity for virtual host controller.

>Manual testing shows native sim works fine without this config, however it breaks nRF clock drivers in bsim

Then, you cannot use it with bsim.

Reply:
> It changes minimum available timing granularity for virtual host controller.

So it is essential for USBIP to work, and the value of 32768 is not fully correct (however it works)? Could this issue be resolved or worked around, or the only proper way would be to emulate Nordic and/or DWC2 USB controllers as simulated bsim devices? That does not sound like a feasible solution